legal-copy/legal-templates.js

```diff
@@ -0,0 +1,62 @@
+module.exports.templateBegin = `
```

Comment:
Maybe worth a follow-up issue, but is there a way we can share this template with the rest of the site so it doesn't inevitably get out of date when we change the site again and forget this is here? (i.e. DRY out the templates)